docs/community-support.md

```diff
@@ -0,0 +1,70 @@
+# Community support
+
+For bugs, you can use the GitHub [issue tracker](https://github.com/prometheus-operator/kube-prometheus/issues/new/choose).
+
+For questions, you can use the GitHub [discussions forum](https://github.com/prometheus-operator/kube-prometheus/discussions).
+
+Many of the `kube-prometheus` project's contributors and users can also be found on the #prometheus-operator channel of the Kubernetes Slack.
+
+`kube-prometheus` is the aggregation of many projects that all have different
+channels to reach out for help and support. While this community strives at
+supporting all users, it is sometimes more efficient to reach out to the
```

Comment:
Maybe something like this:

> While this community strives at supporting all users, and you should never be afraid of asking us first.

As we mentioned that we don't want to discourage folks and majority of the projects we are maintainers of anyways.

Reply:
agreed, I've rephrased.